### PR TITLE
feat: APPS-2459 Add Multiple Components

### PR DESCRIPTION
**Pull request by tinuola**
Connected to [APPS-2459](https://jira.library.ucla.edu/browse/APPS-2459)

**Component Created:**
- MastheadPrimary.vue
- MastheadSecondary.vue
- SectionCardsWithIllustrations.vue
- ResponsiveImage.vue
- SectionDualMasonry.vue
- SectionSpacesList.vue
- SimpleCards.vue

**Stories:**
- ~/stories/MastheadPrimary.stories.js
- ~/stories/MastheadSecondary.stories.js
- ~/stories/SectionCardsWithIllustrations.stories.js
- ~/stories/ResponsiveImage.stories.js
- ~/stories/SectionDualMasonry.stories.js
- ~/stories/SectionSpacesList.stories.js
- ~/stories/SimpleCards.stories.js

**Spec:**
- ~/stories/MastheadPrimary.spec.js
- ~/stories/MastheadSecondary.spec.js
- ~/stories/SectionCardsWithIllustrations.spec.js
- ~/stories/ResponsiveImage.spec.js
- ~/stories/SectionDualMasonry.spec.js
- ~/stories/SectionSpacesList.spec.js
- ~/stories/SimpleCards.spec.js

**Notes:**

MastHeadPrimary.vue and MastHeadSecondary.stories.js are currently missing the SearchHome component per ticket criteria.

**Checklist:**

-   [x] I checked that it is working locally in the dev server
-   [x] I checked that it is working locally in the storybook
-   [ ] I checked that it is working locally in the 
library-website-nuxt dev server
-   [ ] I added a screenshot of it working
-   [ ] UX has reviewed and approved this
-   [x] I assigned this PR to someone on the dev team to review
-   [x] I used a conventional commit message
-   [x] I assigned myself to this PR

[APPS-2459]: https://uclalibrary.atlassian.net/browse/APPS-2459?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/lib-components/MastheadSecondary.vue [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+  // Variants
+  &.has-molecule:before {
+      content: "";
+      position: absolute;
+      top: 0;
+      left: 0;
+      bottom: 0;
+      right: 0;
+      background-image: url("node_modules/ucla-library-design-tokens/assets/svgs/molecule-masthead.svg");
```
Comment (pghorpade) [inline src/lib-components/MastheadSecondary.vue:138]: update this to  background-image: url("ucla-library-design-tokens/assets/svgs/molecule-masthead.svg");

File: src/lib-components/BlockEvent.vue [hunk 2]
```diff
@@ -84,111 +84,106 @@ export default {
...
+        flex-wrap: nowrap;
+        justify-content: center;
+        align-content: flex-start;
+        align-items: flex-start;
     }
 
-    &.color-help {
-      --color-theme: var(--color-help-green-03);
+    .molecule {
+        background-image: url("node_modules/ucla-library-design-tokens/assets/svgs/molecule-placeholder.svg");
```
Comment (pghorpade) [inline src/lib-components/BlockEvent.vue:109]: update this line to background-image: url("ucla-library-design-tokens/assets/svgs/molecule-placeholder.svg");

**Review by pghorpade — CHANGES_REQUESTED** [review]

**Review by pghorpade — CHANGES_REQUESTED** [review]

File: src/lib-components/MastheadSecondary.vue (hunk 1 above)
Comment (tinuola) [reply src/lib-components/MastheadSecondary.vue:138]: Good catch. 👍🏽  Will watch out for this pattern going forward.

**Comment by tinuola** [comment]
@pghorpade Changes addressed. PR ready for a second look. Thanks.

**Review by pghorpade — APPROVED** [review]
👍

**Pull request merged**